### DISCUSSION TITLE
Welcome to HELICS Discussions!

### DISCUSSION THREAD

**Discussion by nightlark** [GMLC-TDC/HELICS#1565] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AmitHREDIITR** [comment]
Hello Everyone
Myself working on HELICS.... my question is can i connect two fedrates in HELICS where one fedrate have Python based and another fedration have C++ based.

**Reply by phlptp** [reply]
Yes, the languages all can work together, and has been done many times.   Can have some in python, some in Matlab, some in C, some in C++, and can even be on different machines.

**Reply by AmitHREDIITR** [reply]
Ok Thanks a lot

**Reply by AmitHREDIITR** [reply]
Hello.. when i am trying to run unmaintained example of python delayed pi-exchange.  it is not working.. please suggest how i can run this delayed example

**Reply by trevorhardy** [reply]
Can you be more specific? What error message were you getting? Were the results not as expected?

**Reply by trevorhardy** [reply]
Also, if you're looking for examples on how to use HELICS I would recommend the [User Guide](https://docs.helics.org/en/latest/user-guide/helics_user_tutorial.html) which links to a [bunch of maintained examples](https://docs.helics.org/en/latest/user-guide/examples/examples_index.html).

**Reply by trevorhardy** [reply]
In my testing, there does appear to be a bug related to filters; [I've filed a bug report.](https://github.com/GMLC-TDC/HELICS/issues/2524)

**Reply by AmitHREDIITR** [reply]
Ok sir thankyou for your suggestions
when i was running this unmaintained example of python delayed pi-exchange. it is not working and was not running after this point.

![image](https://user-images.githubusercontent.com/99208174/226418049-3a9ab6b0-6b96-49b3-a2b4-f822f78a1768.png)

**Reply by trevorhardy** [reply]
OK, that's what I was seeing as well.
